source/javascripts/components/GuidedOnboarding/GuidedOnboarding.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import Collapsible from "./Collapsible/Collapsible";
+import { GuidedOnboardingContent } from "./GuidedOnboardingContent/GuidedOnboardingContent";
+
+interface GuidedOnboardingProps {
+    isEnabled?: boolean;
+}
+
+export const GuidedOnboarding: React.FC<GuidedOnboardingProps> = ({isEnabled = false}) => {
+  return isEnabled ?
```

Comment:
on small screen sizes the content does not fit

![image](https://user-images.githubusercontent.com/3769956/193804778-bf4f31ff-2f3a-4dff-9f35-d0b2f3ed3dfd.png)

Reply:
added resize observer and flex wrap to handle small screen size